docs/arf.md

```diff
@@ -2352,6 +2410,8 @@ Notes:
     attestation and verifies the value encoded at the bit position given
     by the index value in the attestation.
 
+    [**This is problematic and contravenes 16a of the Regulation "16. The technical framework of the European Digital Identity Wallet shall: (a) not allow providers of electronic attestations of attributes or any other party, after the issuance of the attestation of attributes, to obtain data that allows transactions or user behaviour to be tracked, linked or correlated, or knowledge of transactions or user behaviour to be otherwise obtained, unless explicitly authorised by the user;" . An index value to a revocation status list is a unique identifier for a specific credential belonging to a specific user. Once a 3rd party (eg a relying party) has this index value, they can collude with other relying parties who will see the same index value each time attributes from this attestation are presented. They will be able to build up a profile of that user's activity. Given that the user won't have a choice about not presenting this index value, they are effectively sharing a unique identifier with everyone to whom they present any attributes from this attestation, EVEN IF they are using selective disclosure. Assuming selective disclosure is used, relying parties who each receive a small selection of attributes from the user will be able to assemble them into a much more comprehensive set by using this index value as the key. Additionally, once a 3rd party has this index value and the URL of the status list, the 3rd party can watch this revocation status bit constantly to see if it changes even if their relationship with the user has lapsed. This is a privacy issue as any relying party can continue to check if my driving license has been revoked long after my original transaction with them. The could, for example, then blackmail me by threatening to tell friends/colleagues/my employer that my driving license has been revoked. Or that my membership of a club has been revoked. Or that I have lost my job because my employee ID attestation has been revoked. The use of status lists should therefore be avoided unless the correlation and ongoing access issues are handled well enough to satisfy the regulation 16a.**]
+
```

Comment:
This privacy risk is actively being discussed with Member States at the moment. Requirements to mitigate it will be added to ARF 1.7. Basically, these mitigations come down to regular re-issuing of attestations, such that an index value (or any other static identifier in an attestation) cannot be used to link attestation presentations. In addition, specifically for revocation, the ARF already describes that Attestation Providers can use short-lived attestations that do not have to be revoked.